spec/features/h3_object_creation_spec.rb

```diff
@@ -115,5 +117,40 @@
                                                   workflow: 'accessionWF',
                                                   workflow_retry_text: 'Error: shelve : problem with shelve',
                                                   retry_wait: 10)
+    expect(page).to have_text('changing abstract (Public version 1)') # Argo still on user version 1 since only metadata changed
+    expect(page).to have_no_text('Public version 2') # and no user version 2
+    expect_text_on_purl_page(druid: work_druid, text: 'Version 1') # PURL also still only shows Version 1
+    do_not_expect_text_on_purl_page(druid: work_druid, text: 'Version 2') # and no user version 2
```

Comment:
adds some expectations to the current object version created in the spec, which is a metadata only change